src/app/admin/requests/request-details/request-details.component.html

```diff
@@ -1,22 +1,295 @@
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="content">
+    <div class="fieldset">Detalii Cerere</div>
+
+    <div class="fields_row">
+      <div class="field" fxFlex fxLayout="column">
+        <div class="title">Are nevoie de</div>
+        <div class="value">
+          <mat-select
+            formControlName="need"
+            style="width: 230px"
+            placeholder="Alege"
+          >
+            <mat-option
+              *ngFor="let need of needs | keyvalue: enumUnsorted"
+              [value]="need.key"
+            >
+              <img
+                src="/assets/icons/beneficiary/beneficiary_needs/{{
+                  [need.key]
+                }}.svg"
+                width="24"
+                height="24"
+              />
+              {{ need.value }}
+            </mat-option>
+          </mat-select>
+          <mat-error *ngIf="form.get('need').invalid && form.touched">
+            atenție, eroare - cîmp obligatoriu!
+          </mat-error>
+        </div>
+      </div>
+
+      <div class="field" fxFlex fxLayout="column">
+        <div class="title">Boală contagioasă</div>
+        <div class="value">
+          <mat-select
+            formControlName="ilness"
+            style="width: 230px"
+            placeholder="Alege"
+          >
+            <mat-option [value]="false">
+              <mat-icon></mat-icon>
+              Nu are
+            </mat-option>
+            <mat-option [value]="true">
+              <img
+                src="/assets/icons/beneficiary/mask.svg"
+                width="24"
+                height="24"
+              />
+              Are simptome
+            </mat-option>
+          </mat-select>
+          <mat-error *ngIf="form.get('ilness').invalid && form.touched">
+            atenție, eroare - cîmp obligatoriu!
+          </mat-error>
+        </div>
+      </div>
+    </div>
+
+    <div class="fields_row">
+      <div class="field" fxFlex fxLayout="column">
+        <div class="title">Comentarii operator</div>
+        <div class="value">
+          <input
+            matInput
+            formControlName="comments"
+            style="width: 470px"
+            placeholder="---"
+          />
+        </div>
+      </div>
+    </div>
+
+    <div class="fields_row">
+      <div class="field" fxFlex fxLayout="column">
+        <div class="title">Parola secretă</div>
+        <div class="value">
+          <input matInput formControlName="password" style="width: 150px" />
+          <mat-error *ngIf="form.get('password').invalid && form.touched">
+            atenție, eroare - cîmp obligatoriu!
+          </mat-error>
+        </div>
+      </div>
+
+      <div class="field" fxFlex fxLayout="column">
+        <div class="slide-toggle" [ngStyle]="getUrgentStyleObject()">
+          <mat-slide-toggle formControlName="urgent">Urgent?</mat-slide-toggle>
+        </div>
+      </div>
+    </div>
+  </div>
 
-<app-request-form
```

Comment:
`app-request-form` usage was removed, but the component was not removed, remained a lot of dead code

Reply:
This component can be used in case of view existing request. Please create a new issue and I will make necessary adjustments

Reply:
now we start re-engineer the code where <app-request-form> will be used for both new record and existing one - so it will remain ...